dll/win32/shell32/shellrecyclebin/recyclebin_v5_enumerator.c

```diff
@@ -17,6 +18,12 @@ struct RecycleBin5File
     WCHAR FullName[ANY_SIZE];
 };
 
+EXTERN_C HRESULT WINAPI SHCreateFileExtractIconW(LPCWSTR pszPath, DWORD dwFileAttributes, REFIID riid, void **ppv);
```

Comment:
PR #4223 has been committed, so you can remove the line now:
```suggestion
```

Reply:
Done